scss/civicrm/administer/settings/_paymentprocessor.scss

```diff
@@ -31,6 +31,10 @@
           box-sizing: border-box;
           width: 100%;
         }
+
+        .status.no-popup {
+          margin: 0 20px 20px;
```

Comment:
why is this margin applied only here? shouldn't the margin for these status bars be the same everywhere?

Reply:
.status can have margin different based on it's parents' padding or margins,
See some of the variation
https://github.com/civicrm/org.civicrm.shoreditch/pull/177/files/7a76c848c9580a7425848169c3f153b5ad3be2f6#diff-dd79d4af05112df7d44e343ee70b17ecR16
https://github.com/civicrm/org.civicrm.shoreditch/pull/177/files/7a76c848c9580a7425848169c3f153b5ad3be2f6#diff-452398b2bf44710ab6ce3cde7bff0807R4

Reply:
can you show me some screenshots of these different variations?

Reply:
**Delete Payment processor**
![image](https://user-images.githubusercontent.com/26058635/37084335-3c07909e-2218-11e8-8494-8bf6c3bedccb.png)

**Delete Task page**
![image](https://user-images.githubusercontent.com/26058635/37083557-df820ffe-2215-11e8-8a0a-6b182a7c2916.png)

**Print Label task page**
![image](https://user-images.githubusercontent.com/26058635/37083514-bf65774c-2215-11e8-9ec6-85cac2d471fe.png)

**Manage events no margin needed here**
![image](https://user-images.githubusercontent.com/26058635/37084416-778a9878-2218-11e8-8d93-df2ef6139561.png)

**Popups : gets handled due to popup padding**
![image](https://user-images.githubusercontent.com/26058635/37084490-acf089dc-2218-11e8-8951-bf345c62d9a6.png)

**Empty page, No margin needed**
![image](https://user-images.githubusercontent.com/26058635/37084511-be4fd462-2218-11e8-878e-7f737f807f3e.png)

Reply:
ok @mukeshcompucorp wouldn't it be better though, both visually and technically, to have just one type of margin that works in any circumstance (say, a status bar always have a 10px top and bottom margin)? Or would that break the layout?

Reply:
Yes, that breaks different pages.
By standard when contained in a block it should be having `margin: 20px` around it. 
so, we are adding it based on the space it needs, 0 (in case parent element is having that space), 20px or 15px (if parent has 5px already)